src/issue/screens/new-issue.screen.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { ScrollView, StyleSheet, TextInput } from 'react-native';
+
+import {
+  ViewContainer,
+  SectionList,
+} from 'components';
+import { colors, fonts } from 'config';
+import { submitNewIssue } from '../issue.action';
+
+const styles = StyleSheet.create({
+  listItemTitle: {
+    color: colors.black,
+    ...fonts.fontPrimary,
+  },
+  closeActionTitle: {
+    color: colors.red,
+    ...fonts.fontPrimary,
+  },
+  openActionTitle: {
+    color: colors.green,
+    ...fonts.fontPrimary,
+  },
+});
+
+const mapStateToProps = state => ({
+  repository: state.repository.repository,
+});
+
+const mapDispatchToProps = dispatch => ({
+  submitNewIssueByDispatch: (
```

Comment:
Looks like all you really have left to do is map this action to the screen :)

Similarly to `PullMergeScreen`, what we can do is just have a `ListItem` after the `Section Lists` called `Create New Issue` and clicking it will either:

1. Call `submitNewIssueByDispatch` and after it's completed (similarly to [mergePull](https://github.com/gitpoint/git-point/blob/master/src/issue/screens/pull-merge.screen.js#L128-L130), call navigation.navigate to `Issue` with the issue response returned from the `submitNewIssue` call.
2. Call `submitNewIssueByDispatch` and **within**  `submitNewIssue` right after the fetch call is made, do the navigation

I would prefer to have the navigation logic in our components and outside of our action files (option 1) _but_ if it's easier to have the navigation logic within the action method itself I'm okay with it :)